src/m.ts

```diff
@@ -0,0 +1,1294 @@
+            name: "username",
+            description: "Username",
+          },
+        },
+      ],
+    },
+    {
+      name: "volume",
+      description: "Manage volume level",
+      subcommands: [
+        {
+          name: "mute",
+          description: "Set mute",
+        },
+        {
+          name: "unmute",
+          description: "Unset mute",
+        },
+        {
+          name: "ismute",
+          description: "Check the volume status",
+        },
+      ],
+      args: {
+        name: "volume",
+        description: "Volume level up/down/(+/-)number",
+        isOptional: true,
+        suggestions: [
+          {
+            name: "up",
+            description: "Increase the volume by 6.25",
+          },
+          {
+            name: "down",
+            description: "Decrease the volume by 6.25",
+          },
+        ],
+      },
+    },
+    {
+      name: "vpn",
+      description: "Manage VPN connections",
+      subcommands: [
+        {
+          name: "ls",
+          description: "List VPN connections",
+        },
+        {
+          name: "start",
+          description: "Start a VPN connection",
+          args: [
+            {
+              name: "VPN",
+              description: "VPN connection name",
+              isOptional: true,
+            },
+            {
+              name: "USER",
+              description: "VPN connection user",
+              isOptional: true,
+            },
+            {
+              name: "PASS",
+              description: "VPN connection password",
+              isOptional: true,
+            },
+            {
+              name: "SECRET",
+              description: "VPN connection secret",
+              isOptional: true,
+            },
+          ],
+        },
+        {
+          name: "stop",
+          description: "Stop a VPN connection",
+          args: {
+            name: "VPN",
+            description: "VPN connection name",
+            isOptional: true,
+          },
+        },
+        {
+          name: "status",
+          description: "Get the status of a VPN connection",
+          args: {
+            name: "VPN",
+            description: "VPN connection name",
+            isOptional: true,
+          },
+        },
+      ],
+    },
+    {
+      name: "wallpaper",
+      description: "Change wallpaper",
+      options: [
+        {
+          name: "",
+          description: "Path to the wallpaper",
+        },
+      ],
+    },
+    {
+      name: "wifi",
+      description: "Manage wifi",
+      subcommands: [
+        {
+          name: "status",
+          description: "Get wifi status",
+        },
+        {
+          name: "scan",
+          description: "Scan wifi",
+        },
+        {
+          name: "showpassword",
+          description: "Show wifi network password",
+          args: {
+            name: "ESSID",
+            description: "ESSID",
+            isOptional: true,
+          },
```

Comment:
We'll reuse the generator from [wifi-password](https://github.com/withfig/autocomplete/blob/68a357150d28094967cb9e963cfbb2640605ca05/src/wifi-password.ts)

```ts
const generateWifiNetworks: Fig.Generator = generators: {
  script:
    "networksetup -listallhardwareports | awk '/Wi-Fi/{getline; print $2}' | xargs networksetup -listpreferredwirelessnetworks | tail -n +2",
  postProcess: (out) =>
    out
      .trim()
      .split("\n")
      .map((line) => ({ name: line.trim() })),
},
```

```suggestion
          args: {
            name: "ESSID",
            description: "ESSID",
            isOptional: true,
            generators: generateWifiNetworks,
          },
```

Can use the same generator for any other place that takes an ssid